### DISCUSSION TITLE
Ritardo audio..

### DISCUSSION THREAD

**Discussion by Roscopct** [AndreyPavlenko/Fermata#523] [Announcements]
Buongiorno.. ho perso le impostazioni e non ricordo più a quale valore avevo impostato il ritardo audio.. c'è un valore più o meno di default??
Grz

**Comment by AndreyPavlenko** [comment]
It very much depends on the device.

**Comment by Roscopct** [comment]
Sapevo che il ritardo audio funzionava solo con motore Vlc e non su Youtube.. è ancora così?? Con Youtube come si può risolvere?? Grz

**Reply by AndreyPavlenko** [reply]
There is no such option for youtube.

**Reply by Roscopct** [reply]
Grazie Andrey

**Comment by Roscopct** [comment]
Può servire attivare la funzione dentro android auto
" attiva dump di latenza audio"???
